newIDE/app/src/EventsFunctionsExtensionsLoader/MetadataDeclarationHelpers.js

```diff
@@ -830,6 +830,85 @@ const getStringifiedExtraInfo = (property: gdPropertyDescriptor) => {
     : '';
 };
 
+const uncapitalizedFirstLetter = (string: string): string =>
+  string.length < 1
```

Comment:
You could use the condition `!string` instead but I'm not sure it's easier to read.